packages/web-client/app/components/card-pay/merchant-payment-request-card/index.hbs

```diff
@@ -16,10 +16,25 @@
       rel="noopener noreferrer"
       data-test-payment-request-beta-access-link
     >
-        Download TestFlight
-        {{svg-jar "caret-right-large" class="merchant-payment-request-card__testflight-link-icon"}}
+      Download TestFlight
+      {{svg-jar "caret-right-large" class="merchant-payment-request-card__testflight-link-icon"}}
     </a>
-   </div>
+  </div>
+  <div class="merchant-payment-request-card__section merchant-payment-request-card__get-card-wallet--mobile">
+    <a
+      class="merchant-payment-request-card__card-wallet"
+      href={{this.testFlightLink}}
+      target="_blank"
+      rel="noopener noreferrer"
+      data-test-payment-request-beta-access-link
+    >
+      {{svg-jar "card-wallet-app-icon" class="merchant-payment-request-card__card-wallet-icon" width="40" height="40"}}
+      <span class="merchant-payment-request-card__card-wallet-text">
+        Card Wallet
+      </span>
+      Download TestFlight
+    </a>
+  </div>
   <CardPay::MerchantPaymentRequestCard::PaymentRequest
     class="merchant-payment-request-card__section"
     @paymentURL={{@paymentURL}}
```

Comment:
![image](https://user-images.githubusercontent.com/39579264/134181841-c75debce-4c66-4400-a37b-132aa300d77a.png)

Reply:
Major change here is the "Card Wallet | Download TestFlight" / download prompt on a smaller screen. The whole of the top section is a clickable link.